app/views/application/_email-confirmation-reminder.html.erb

```diff
@@ -0,0 +1,6 @@
+<section class="email-reminder" aria-label="Notice" role="region">
+  <p class="govuk-body govuk-!-margin-0">
```

Comment:
What are the rules about when it's cool to use an inline class like this and when we should have it in our CSS?

Reply:
The `govuk-!` classes are used for overriding existing styles. In this case we're overriding the margin that's provided by the `govuk-body` class, so it's an appropriate use. Otherwise I'd have to effectively duplicate the override class in our own styles, which would be a waste of CSS.

Reply:
gotcha, thanks!